plasmapy/physics/parameters.py

```diff
@@ -778,11 +776,15 @@ def gyrofrequency(B, particle='e-', signed=False, Z=None):
     return omega_ci
 
 
-@utils.check_quantity({'B': {'units': u.T},
-                       'Vperp': {'units': u.m / u.s, 'can_be_nan': True},
-                       'T_i': {'units': u.K, 'can_be_nan': True},
-                       })
-def gyroradius(B, particle='e-', *, Vperp=np.nan * u.m / u.s, T_i=np.nan * u.K):
+@utils.check_quantity(B={'units': u.T},
+                      Vperp={'units': u.m / u.s, 'can_be_nan': True},
+                      T_i={'units': u.K, 'can_be_nan': True},
+                      )
+def gyroradius(B: u.T,
+               particle='e-',
+               *,
```

Comment:
Not a review comment or anything but I am curious what `*` does here!

Reply:
It's a design decision I'm still queasy about, but we sort of . By adding *, you make sure the following arguments can only be input as keyword arguments - thus, one cannot mistake them.

It's weird, I know.